index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Akhror Valiev</title>
+
+	<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1" />
+
+	<link href="https://fonts.googleapis.com/css2?family=Russo+One&display=swap" rel="stylesheet">
+
+	<link href="https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@500&family=Russo+One&display=swap" rel="stylesheet">
+
+	<link rel="stylesheet" type="text/css" href="default.css">
+
+</head>
+<body>
+
+	<section class="s1">
+		<div class="main-container">
+			<div class="greeting-wrapper">
+				<h1>Hi, I'm Akhror Valiev</h1>
+			</div>
+
+
+			
+			<div class="intro-wrapper">
+				
+				<div class="nav-wrapper">
+					<div class="dots-wrapper">
+						<div id="dot-1" class="browser-dot"></div>
+						<div id="dot-2" class="browser-dot"></div>
+						<div id="dot-3" class="browser-dot"></div>
+					</div>
+
+					<ul id="navigation">
+						<li><a href="#">Contact</a></li>
+						
+					</ul>
+				</div>
+
+				<div class="left-column">
+					<img id="profile_pic" src="images/pic.jpg">
+					<h5 style="text-align: center; Line-height: 0;">Personilze Theme</h5>
+
+
+					<div id="theme-options-wrapper">
+						<div id="light-mode" class="theme-dot"></div>
+						<div id="green-mode" class="theme-dot"></div>
+						<div id="purple-mode" class="theme-dot"></div>
+						<div id="blue-mode" class="theme-dot"></div>  
+					</div>
+
+					<p id="settings-note">These settings will be saved for<br>your next visit</p>
+				</div>
+
+				<div class="right-column">
+
+					<div id="preview-shadow">
+						<div id="preview">
+							<div id="corner-tl" class="corner"></div>
+							<div id="corner-tr" class="corner"></div>
+							<h3>what I Do</h3>
+							<p>I was a lead developer in past life, now I enjoy testing courses.</p>
+							<div id="corner-br" class="corner"></div>
+							<div id="corner-bl" class="corner"></div>
+						</div>
+					</div>
+				</div>
+
+			</div>
+				
+		</div>
+	</section>
+
+	
+	<section class="s2">
+		<div class="main-container">
+			About
+		</div>
+	</section>
+
+	<section class="s1">
+		<div class="main-container">
+			Portfolio
+		</div>
+	</section>
+
+	<section class="s2">
+		<div class="main-container">
+			Contact
+		</div>
+	</section>
+
```

Comment:
end of the body, use a footer tag that makes your website well-structured and semantic.

Reply:
Great work @akhror-valiev. You made a very good review here with excellent explanation. Also kuddos on using the github review feature. I suggest you make a change however to how your comments are phrased:

- [ ] Currently your comments feel like orders:

This is an example of how comments can be phrased:
* This is a polite suggestion: *"I think it would be better to use semantic tags"*
This is a requirement for you to successfully get approved:
<img width="1660" alt="Screenshot 2022-09-26 at 12 59 35" src="https://user-images.githubusercontent.com/48546992/192269613-7a1f4f92-5375-4ffe-b461-9df4fd5fd421.png">